.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "${workspaceFolder}/env/bin/python"
```

Comment:
Did you mean to commit this?

Reply:
Nope, thanks. Removed it and added it to the .gitignore.